src/Codeception/Module/Yii2.php

```diff
@@ -482,18 +483,19 @@ public function amLoggedInAs($user)
      * ```
      * instead of calling `haveFixtures` in Cest `_before`
      *
-     * @param $fixtures
+     * @param array $fixtures
+     * @param string|null $owner
      * @part fixtures
      */
-    public function haveFixtures($fixtures)
+    public function haveFixtures($fixtures, $owner = null)
     {
         if (empty($fixtures)) {
             return;
         }
-        $fixturesStore = new Yii2Connector\FixturesStore($fixtures);
+        $fixturesStore = new Yii2Connector\FixturesStore($fixtures, $owner);
         $fixturesStore->unloadFixtures();
         $fixturesStore->loadFixtures();
-        $this->loadedFixtures[] = $fixturesStore;
+        $this->loadedFixtures[$owner] = $fixturesStore;
```

Comment:
What happens if `$owner` is `null`?

Reply:
Then the key will be '' (an empty string).
So first call to `haveFixtures([...])` will work, but subsequent will not load any fixtures.
That's the problem, yeah

Reply:
I'm not saying it's a problem, but it doesn't seem like something we want... why is `$owner` an optional argument?